static/js/services/playsound.js

```diff
@@ -156,7 +159,17 @@ define(['underscore', 'Howler', 'require'], function(_, Howler, require) {
 	window.PLAYSOUND = registry; // make available for debug.
 
 	// playSound
-	return [function() {
+	return ["appData", function(appData) {
+
+		Sound.prototype.shouldPlaySound = function () {
+			var data = appData.get();
+			var play = true;
+			if (data && data.master.settings.playSoundEffects !== null) {
+				play = data.master.settings.playSoundEffects;
+			}
+			//console.log('shouldPlaySound', play);
+			return play;
+		};
```

Comment:
Moved `Sound.prototype.shouldPlaySound` into the service and am not saving `appData` to a variable.